### PR TITLE
Run go fmt

### PR DESCRIPTION
**Pull request by kalafut**

### CODE REVIEW

File: command/server.go [hunk 1]
```diff
@@ -21,6 +20,8 @@ import (
 	"sync"
 	"time"
 
+	"go.uber.org/atomic"
+
```
Comment (calvn) [inline command/server.go]: Can we remove this newline?

File: helper/testhelpers/teststorage/teststorage.go [hunk 2]
```diff
@@ -2,13 +2,14 @@ package teststorage
...
-	"github.com/hashicorp/vault/helper/testhelpers"
-	"github.com/hashicorp/vault/physical/raft"
 	"io/ioutil"
 	"os"
 	realtesting "testing"
 	"time"
 
+	"github.com/hashicorp/vault/helper/testhelpers"
+	"github.com/hashicorp/vault/physical/raft"
+
```
Comment (calvn) [inline helper/testhelpers/teststorage/teststorage.go]: Extra newline here as well

File: http/handler.go [hunk 3]
```diff
@@ -17,6 +16,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/NYTimes/gziphandler"
+
```
Comment (calvn) [inline http/handler.go]: Extra newline here

File: vault/external_tests/misc/recovery_test.go [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
 
 import (
-	"github.com/go-test/deep"
-	"go.uber.org/atomic"
 	"path"
 	"testing"
 
+	"github.com/go-test/deep"
+	"go.uber.org/atomic"
+
```
Comment (calvn) [inline vault/external_tests/misc/recovery_test.go]: Extra newline

File: vault/ha.go [hunk 5]
```diff
@@ -6,11 +6,12 @@ import (
 	"crypto/x509"
 	"errors"
 	"fmt"
-	"github.com/hashicorp/vault/vault/seal/shamir"
 	"strings"
 	"sync/atomic"
 	"time"
 
+	"github.com/hashicorp/vault/vault/seal/shamir"
+
```
Comment (calvn) [inline vault/ha.go]: Extra newline

File: vault/init_test.go [hunk 6]
```diff
@@ -2,10 +2,11 @@ package vault
 
 import (
 	"context"
-	"github.com/hashicorp/vault/vault/seal"
 	"reflect"
 	"testing"
 
+	"github.com/hashicorp/vault/vault/seal"
+
```
Comment (calvn) [inline vault/init_test.go]: Extra newline

**Comment by kalafut** [comment]
@calvn Cleaned it up a bit.

File: vault/external_tests/identity/identity_test.go [hunk 7]
```diff
@@ -1,9 +1,10 @@
 package identity
 
 import (
+	"testing"
+
 	"github.com/go-ldap/ldap"
 	"github.com/hashicorp/vault/sdk/helper/ldaputil"
-	"testing"
```
Comment (calvn) [inline vault/external_tests/identity/identity_test.go]: Extra newline here

Comment (kalafut) [reply vault/external_tests/identity/identity_test.go]: Fixed!

**Pull request merged**